etcdctl/ctlv2/ctl.go

```diff
@@ -71,7 +76,8 @@ func Start() {
 		command.NewAuthCommands(),
 	}
 
-	err := app.Run(os.Args)
+	err := run(app)
+
```

Comment:
don't add a line here, the call with `err := ...` and the following `if err != nil` are usually grouped together